Files/Views/SettingsPages/Appearance.xaml

```diff
@@ -43,6 +43,16 @@
                     ItemsSource="{Binding Themes}"
                     SelectedIndex="{Binding SelectedThemeIndex, Mode=TwoWay}" />
 
+                <ComboBox
+                    x:Name="ColorSchemeChooser"
+                    Width="Auto"
+                    MinWidth="200"
+                    MaxWidth="250"
+                    Header="Color scheme"
+                    HeaderTemplate="{StaticResource CustomHeaderStyle}"
+                    ItemsSource="{Binding ColorSchemes}"
+                    SelectedItem="{Binding SelectedThemeName, Mode=TwoWay}" />
```

Comment:
Is it posible to only show this combo box when Files detects custom themes in the directory?

Reply:
Ye